pages/sequence-files/[id].js

```diff
@@ -71,6 +84,42 @@ export default function SequenceFile({
                     </DataItemValue>
                   </>
                 )}
+                {sequenceFile.flowcell_id && (
+                  <>
+                    <DataItemLabel>Flowcell ID</DataItemLabel>
+                    <DataItemValue>{sequenceFile.flowcell_id}</DataItemValue>
+                  </>
+                )}
+                {truthyOrZero(sequenceFile.lane) && (
```

Comment:
This works fine. Looks like `lane` has a minimum value of 1 though, so just using `sequenceFile.lane` without the `truthyOrZero` also works fine. But up to you if you want to change it. I suppose if the schema ever allows a zero value, your method would be more robust, but I doubt we’ll ever have a lane=0?

Reply:
Changed, I am also doubtful we’ll ever have a lane=0.